### PR TITLE
feature/PEER-433: Home page updates - community focused

### PR DESCRIPTION
**Pull request by AweAwe95**

### CODE REVIEW

File: app/components/HomePage/ActionsSection/ActionsSection.tsx [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+import screen_3 from 'images/scr_3.jpg';
+import screen_4 from 'images/scr_4.jpg';
+import messages from '../messages';
+
+import { styles } from './ActionsSection.styled';
+import { pageStyles } from '../HomePage.styled';
+
+const ActionsSection: React.FC = (): JSX.Element => {
+  const [startFirstActionAnimation, setStartFirstActionAnimation] =
+    useState<boolean>(false);
```
Comment (Lampardks) [inline app/components/HomePage/ActionsSection/ActionsSection.tsx:15]: why variables to start animation?

you can use component
```
import { CSSProperties } from 'react'

import { useInView } from 'react-intersection-observer'

type Props = {
  style?: CSSProperties
  className?: string
}

const InViewport: React.FC<Props> = ({
  children,
  style,
  className,
}): JSX.Element => {
  const { ref, inView } = useInView({ triggerOnce: true })
  return (
    <div className={className} style={style} ref={ref}>
      {inView && children}
    </div>
  )
}

export default InViewport
```

File: app/components/HomePage/FormSection/FormField.tsx [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+  meta: any;
+};
+
+const FormField: React.FC<FormFieldProps> = ({
+  input,
+  label,
+  disabled,
+  meta,
+}): JSX.Element => {
+  const [showTip, setShowTip] = useState(true);
```
Comment (Lampardks) [inline app/components/HomePage/FormSection/FormField.tsx:18]: use hook useTrigger

File: app/components/HomePage/Header/Header.tsx [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+import { pageStyles } from '../HomePage.styled';
+import { styles } from './Header.styled';
+
+const Header: React.FC = (): JSX.Element => {
+  const [sticky, setSticky] = useState<boolean>(false);
+
+  useEffect(() => {
+    window.addEventListener('scroll', () => {
+      setSticky(window.scrollY > 0);
+    });
```
Comment (Lampardks) [inline app/components/HomePage/Header/Header.tsx:20]:
```
return () => {
  window.removeEventListener('scroll', func)
}
```

Comment (AweAwe95) [reply app/components/HomePage/Header/Header.tsx:21]: done

File: app/components/HomePage/FormSection/FormField.tsx (hunk 2 above)
Comment (AweAwe95) [reply app/components/HomePage/FormSection/FormField.tsx:18]: done

File: app/components/HomePage/ActionsSection/ActionsSection.tsx (hunk 1 above)
Comment (AweAwe95) [reply app/components/HomePage/ActionsSection/ActionsSection.tsx:16]: -

File: app/utils/animation.js [hunk 4]
```diff
@@ -44,3 +38,36 @@ export function scrollToErrorField(errors) /* istanbul ignore next */ {
   const keys = Object.keys(errors);
   scrollToSection(`#${formatStringToHtmlId(keys[0])}`);
 }
+
+export function scrollTrigger(selector, action, options = {}) {
+  let els = document.querySelectorAll(selector);
```
[the remark this answers is not in the document]
Comment (AweAwe95) [reply app/utils/animation.js]: done

File: app/components/HomePage/ActionsSection/ActionsSection.tsx [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+  const [startSecondActionAnimation, setStartSecondActionAnimation] =
+    useState<boolean>(false);
+  const [startThirdActionAnimation, setStartThirdActionAnimation] =
+    useState<boolean>(false);
+  const [startFourthActionAnimation, setStartFourthActionAnimation] =
+    useState<boolean>(false);
+
+  useEffect(() => {
+    scrollTrigger('.first-action', () => setStartFirstActionAnimation(true));
```
[the remark this answers is not in the document]
Comment (AweAwe95) [reply app/components/HomePage/ActionsSection/ActionsSection.tsx:25]: done

File: app/components/HomePage/ActionsSection/ActionsSection.tsx [hunk 6]
```diff
@@ -0,0 +1,170 @@
+import React, { useEffect, useState } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { scrollTrigger } from 'utils/animation';
+import screen_1 from 'images/scr_1.jpg';
+import screen_2 from 'images/scr_2.jpg';
+import screen_3 from 'images/scr_3.jpg';
+import screen_4 from 'images/scr_4.jpg';
```
[the remark this answers is not in the document]
Comment (AweAwe95) [reply app/components/HomePage/ActionsSection/ActionsSection.tsx]: -

File: app/utils/animation.js [hunk 7]
```diff
@@ -4,19 +4,13 @@ import { HEADER_HEIGHT } from 'containers/Header/constants';
...
-    [pathname],
-  );
-
-  useEffect(
-    () => {
-      setTimeout(scrollToSection, 250);
-    },
-    [hash],
-  );
+  useEffect(() => {
```
[the remark this answers is not in the document]
Comment (AweAwe95) [reply app/utils/animation.js:7]: -

**Review by Lampardks — APPROVED** [review]

**Pull request merged**